preprocessors/geojson-stacker.preprocessor.js

```diff
@@ -0,0 +1,26 @@
+var fs = require('fs');
+var stackerOfWorlds = require('stacker-of-worlds');
+var geojsonhint = require('geojsonhint');
+
+module.exports = function(infile, outfile, callback) {
+  fs.readFile(infile, 'utf8', function(err, data) {
+    if (err) return callback(err);
+    var stacked = stackerOfWorlds.stack(JSON.parse(data));
+    fs.writeFile(outfile, JSON.stringify(stacked), callback);
+  });
+};
+
+module.exports.description = 'Move all longitudes into the +/-180 range';
+
+module.exports.criteria = function(infile, info, callback) {
+  if (info.filetype !== 'geojson') return callback(null, false);
+
+  fs.readFile(infile, 'utf8', function(err, data) {
+    var errors = geojsonhint.hint(data).filter(function(lintError) {
```

Comment:
I do not like this, but I did it out of fear that invalid GeoJSON would cause problems in stacker-of-worlds; and, as mentioned, I don't know the best way to handle those problems.

This introduces other troubles, though. One of the test fixtures, for example, as a `crs` property on a GeoJSON Feature — and geojsonhint errors on that, even though it's no concern of stacker-of-worlds.